src/app/ReviewService.Abstractions/ReviewSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// The review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettings
+{
+	public int PrimaryActionCompletedCount { get; set; }
+
+	public int SecondaryActionCompletedCount { get; set; }
+
+	public int ApplicationLaunchedCount { get; set; }
+
+	public DateTimeOffset? ApplicationFirstLaunched { get; set; }
```

Comment:
I think this could be non-nullable if you initialize it using `DateTimeOffset.Now`.
You might need a constructor for this.